Makefile

```diff
@@ -24,10 +29,12 @@ assets:
 	npm cache clear && \
 	npm install && \
 	cd ${CWD}
-	python manage.py webpack --extra=--display-error-details
+	${POOTLE_CMD} webpack --extra=--display-error-details
 	mkdir -p ${ASSETS_DIR}
-	python manage.py collectstatic --noinput --clear -i node_modules -i .tox -i docs ${TAIL}
-	python manage.py assets build ${TAIL}
+
+	${POOTLE_CMD} collectstatic --noinput --clear -i node_modules -i .tox -i docs ${TAIL}
+	${POOTLE_CMD} assets build ${TAIL}
+
 	chmod 664 ${ASSETS_DIR}.webassets-cache/*
 
 travis-assets:
```

Comment:
I'd suggest we change occurances of `python manage.py` here also, to be consistant and not leave vestiges of an approach we'd like to deprecate.